src/inject/dynamic-theme/watch.ts

```diff
@@ -37,24 +37,24 @@ function collectUndefinedElements(root: ParentNode) {
         });
 }
 
-function customElementsWhenDefined(tag: string) {
+const resolvers = new Map<string, () => void>();
+
+const handleIsDefined = (e: any) => {
+    if (resolvers.has(e.details.tag)) {
+        const resolve = resolvers.get(e.details.tag);
+        resolve && resolve();
+    }
+}
+
+function customElementsWhenDefined($tag: string) {
```

Comment:
Why did it switch from `tag` to `$tag`? Usually `$` is a variable prefix in PHP. Also `$` was used as a global jQuery variable, and there was a convention for a jQuery selector object like `var $elements = $('.some-class').` I sometimes prepend `$` for some encoded strings, or when a variable is later processed into another one without `$`, maybe it is a bad practice.

Reply:
Probably to see the difference in

```typescript
 {detail: {tag: tag}}));
````